pytorch_lightning/utilities/cli.py

```diff
@@ -35,6 +35,9 @@
 else:
     ArgumentParser = object
 
+LRScheduler = (_LRScheduler, ReduceLROnPlateau)
+LRSchedulerType = Union[Type[_LRScheduler], Type[ReduceLROnPlateau]]
```

Comment:
Might be useful to have in https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/utilities/types.py